CC/Sounder/recorder.cc

```diff
@@ -277,7 +283,14 @@ herr_t Recorder::initHDF5(const std::string& hdf5)
         write_attribute(mainGroup, "BS_BEACON_ANT", (int)cfg->beacon_ant);
 
         // Number of antennas on Base Station
-        write_attribute(mainGroup, "BS_NUM_ANT", (int)cfg->getNumAntennas());
+        write_attribute(mainGroup, "BS_NUM_ANT", (int)cfg->getNumAntennas());  // TODO: REMOVE SOON, REPLACE BY PER CELL
+
+	// Number of antennas on Base Station (per cell)
+        std::vector<std::string> bs_ant_num_per_cell(cfg->bs_sdr_ids.size());
+        for (size_t i = 0; i < bs_ant_num_per_cell.size(); ++i) {
+            bs_ant_num_per_cell[i] = std::to_string(cfg->bs_sdr_ids[i].size() * (int)cfg->bsChannel.length());
+        }
+        write_attribute(mainGroup, "BS_ANT_NUM_PER_CELL", bs_ant_num_per_cell);
```

Comment:
I am curious, is this metadata reflected in the `plot_hdf5` tool?

Reply:
not yet. not needed until we actually have the entire multi-cell framework working. I kept the old field though so it should not break anything